### ISSUE TITLE
The `use-dictionary-union` refactoring should not be suggested when projects support Python < 3.9

### ISSUE THREAD

**Issue by br3ndonland** [sourcery-ai/sourcery#223]
### Issue description or question

The dictionary union operator was [introduced in Python 3.9](https://docs.python.org/3/whatsnew/3.9.html). Sourcery provides a [`use-dictionary-union`](https://docs.sourcery.ai/refactorings/use-dictionary-union/) refactoring to suggest the dictionary union operator.

The Sourcery GitHub app is recommending the `use-dictionary-union` refactoring on projects that support Python 3.8 and below.

Example: br3ndonland/fastenv#9

Suggestions:

- Read the supported Python versions from packaging configuration files like _pyproject.toml_ or _setup.py_
- Only suggest `use-dictionary-union` when the minimum supported Python version is 3.9.

### Sourcery Version

Whatever is in the GitHub app

### Code editor or IDE name and version

GitHub app

### OS name and version

Linux: Ubuntu 20.04 (GitHub Actions)

**Comment by Hellebore** [comment]
Hi - apologies for this. I think it would be better if we didn't reopen the sourcery PR once it has been manually closed.

 You can configure a Python version in a .sourcery.yaml file in your project which should resolve this - see our docs here: https://docs.sourcery.ai/Configuration/Project-Settings/

**Comment by ddelange** [comment]
> I do still think reading the Python version field from project metadata would be a sensible default

Agreed here, I just starred and got a huge PR full of f-string suggestions for a lib that supports py2

**Comment by br3ndonland** [comment]
Closing due to lack of activity.
